### DISCUSSION TITLE
Recommended Hardware

### DISCUSSION THREAD

**Discussion by twister36** [exadel-inc/CompreFace#1181] [General]
I'm looking for guidance for recommended hardware to run CompreFace on. 

Right now, I'm running it on a Windows 10 desktop, running docker... but I would like to get something dedicated for CompreFace... ideally something small like a NCU. What should I look for? My load is small, only a few images processed per hour.

**Comment by renaatdb** [comment]
> NCU

You mean NUC ?

**Comment by renaatdb** [comment]
I have Dell OptiPlex 7050 Micro Tower (Intel Core i5-6500T, 8 GB, 128 GB M.2 SSD) WIndows 10 Pro (Renewed) / USB Coral.
HA on virtual machine.
